addOns/help/src/main/javahelp/contents/releases/2.15.0.html

```diff
@@ -19,28 +19,50 @@ <H1>Release 2.15.0</H1>
 
 <H3>Scripts as First Class Scan Rules</H3>
 
-TODO add more details...
+Active and passive scan rules scripts can now be treated as "first class" scan rules.
+This means that they can be individually referenced in an active scan policy, in the 
+passive scan rules page, and in Automation Framework plans.
+
+In addition directories of scripts can now be added with all of the scripts enabled -
+this will make it much more straightfoward to manage script rules in automation.
 
 <H3>Menu Items Restructured</H3>
 
-TODO add more details...
+The desktop context sensitive menu items have been reordered, and grouped in a more logical way.
+This should make it much easier to find the menu item you want, when you want it.
 
 <H3>Automation Framework GitHub Action</H3>
 
-TODO add more details...
+There is a new ZAP GitHub action - the 
+<a href="https://github.com/marketplace/actions/zap-automation-framework-scan">ZAP Automation Framework Scan</a>.
+
+The Automation Framework provides a great balance between ease of use and flexibility + functionality.
+If you want to perform any non trivial automation with ZAP then the
+<a href="https://www.zaproxy.org/docs/automate/automation-framework/">Automation Framework</a> is probably your best bet.
+
+<H3>New Docker Hub Organisation</H3>
+
+We have a new DockerHub organisation for the ZAP Docker images: https://hub.docker.com/u/zaproxy
+
+We are still using the <a href="https://hub.docker.com/u/softwaresecurityproject">softwaresecurityproject</a> org for 2.15.0 but we will probably not use it for the following releases.
+
+We do recommend that you switch from `softwaresecurityproject` to <a href="https://hub.docker.com/u/zaproxy">zaproxy</a> sooner rather than later.
 
 <h3>Dependency Updates</h3>
 
 As usual the release includes dependency updates.
 <p>
 The following libraries were updated:
 
-TODO update...
-
 <ul>
-  <li>Commons Lang, 3.12.0 → 3.13.0</li>
-  <li>Flatlaf 3.1.1 → 3.2.1</li>
-  <li>RSyntaxTextArea, 3.3.3 → 3.3.4</li>
+  <li>Commons Codex, 1.16.0 → 1.16.1</li>
+  <li>Commons IO, 2.13.0 → 2.16.1</li>
+  <li>Commons Lang3, 3.13.0 → 3.14.0</li>
+  <li>Commons Logging, 1.2 → 1.3.1</li>
+  <li>Commons Text, 1.10.0 → 1.12.0</li>
+  <li>Flatlaf, 3.2.1 → 3.4.1</li>
+  <li>Java Semver, 0.9.0 → 0.10.2</li>
+  <li>Rsyntaxtextarea, 3.3.4 → 3.4.0</li>
 </ul>
 
 The following library was added:
```

Comment:
This one should be removed, it was from previous version.

Reply:
Arg yeah, meant to remove that! Now done.